sandbox/apparmor/notify/notify_test.go

```diff
@@ -385,6 +460,37 @@ func (s *notifySuite) TestRegisterFileDescriptorErrors(c *C) {
 	c.Check(receivedVersion, Equals, notify.ProtocolVersion(0))
 	c.Check(pendingCount, Equals, 0)
 
+	// EACCES during registration
+	ioctlCalls = 0
+	restore = notify.MockIoctl(func(fd uintptr, req notify.IoctlRequest, buf notify.IoctlRequestBuffer) ([]byte, error) {
+		c.Assert(fd, Equals, fakeFD)
+
+		ioctlCalls++
+
+		// First expect check for version 7, then for version 3
+		switch ioctlCalls {
+		case 1:
+			c.Check(req, Equals, notify.APPARMOR_NOTIF_REGISTER)
+			// Expect listener ID 0, set arbitrary ID/ready/pending
+			respBuf := checkIoctlBufferRegister(c, buf, notify.ProtocolVersion(7), 0, 123)
+			// On v7, return an error on the APPARMOR_NOTIF_REGISTER
+			return respBuf, fmt.Errorf("cannot perform IOCTL request %v: %w (%s)", req, unix.EINVAL, unix.ErrnoName(unix.EACCES))
```

Comment:
```suggestion
			return respBuf, fmt.Errorf("cannot perform IOCTL request %v: %w (%s)", req, unix.EACCES, unix.ErrnoName(unix.EACCES))
```

I think changing this also requires changing some other assertions.